dictionaryutils/__init__.py

```diff
@@ -82,7 +83,12 @@ def dump_schemas_from_dir(directory):
     """Dump schema as a json"""
 
     with visit_directory(directory):
-        return {path: load_yaml(path) for path in glob.glob('*.yaml')}
+        result = {path: load_yaml(path) for path in glob.glob('*.yaml')}
+        if not '_settings.yaml' in result:
+            result['_settings.yaml'] = {}
+        result['_settings.yaml']['_dict_commit'] = DICTCOMMIT
```

Comment:
When will the `dictionaryutils/version_data.py` file be modified? I'm not familiar with the interaction between these utils and the dictionary. When/where does the `version_data.py` get populated with actual info? Looks like this was previously done in sheepdog: https://github.com/uc-cdis/sheepdog/pull/156/files#diff-3254677a7917c6c01f55212f86c57fbfL44

Reply:
I believe this code is only used by dump_schema.py which gets called by each of our dictionaries via run_tests.sh (https://github.com/uc-cdis/dictionaryutils/blob/master/run_tests.sh#L5)

https://github.com/uc-cdis/datadictionary/blob/develop/.travis.yml#L6

Reply:
And correct, this code was originally in sheepdog, but when we switched from having versions of sheepdog per dictionary to pulling the dictionary from S3 we never fixed the version information loading. So we are doing that now but need to include it in the schema.json that gets loaded somewhere so we are doing that here

Reply:
So it's expected that the dictionary in s3 will have this `_settings.yaml` with the `_dict*` info already populated? This code is going to always replace it with an empty string so the schemas that get dumped won't have that information. Not sure if the intention is to default to an empty string or not

Reply:
oh. wait. Looks like run_tests gets the information and populates the version_data file?

Reply:
Yep. run_tests gets it.